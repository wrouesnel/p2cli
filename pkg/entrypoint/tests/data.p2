{{simple_value1}}
{{simple_value2}}
