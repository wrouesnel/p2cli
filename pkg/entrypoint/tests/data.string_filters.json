{
  "s": "text to convert a few times"
}