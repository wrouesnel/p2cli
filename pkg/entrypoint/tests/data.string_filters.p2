string
{{ s | string | bytes | string }}

bytes
{{ s | bytes | string | bytes | string }}