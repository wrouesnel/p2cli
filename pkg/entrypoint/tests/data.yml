simple_value1: a value
simple_value2: a value
