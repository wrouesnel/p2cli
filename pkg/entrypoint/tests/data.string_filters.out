string
text to convert a few times

bytes
text to convert a few times